requests/__init__.py

```diff
@@ -56,7 +56,7 @@
 # See https://github.com/kennethreitz/requests/issues/3213 as well.
 import ssl
 
-if ssl.OPENSSL_VERSION_INFO < (1, 0, 1):
+if hasattr(ssl, 'OPENSSL_VERSION_INFO') and getattr(ssl, 'OPENSSL_VERSION_INFO') < (1, 0, 1):
```

Comment:
@sigmavirus24 I didn't mean to come off as hostile at all, apologies for that. I didn't think anyone else was calling me dense, I thought that was myself just _being_ dense, which I would like to retain the right to insult myself (only) 😉 . Thank you for your feedback, still trying to get a handle on my tone when contributing to FOSS, will adjust.